.evergreen/run-serverless-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+source ./.evergreen/env.sh
+
+FEATURE_FLAGS="zstd-compression,snappy-compression,zlib-compression"
+DEFAULT_FEATURES=""
+
+if [ "$ASYNC_RUNTIME" = "async-std" ]; then
+    FEATURE_FLAGS="${FEATURE_FLAGS},async-std-runtime"
+    DEFAULT_FEATURES="--no-default-features"
+elif [ "$ASYNC_RUNTIME" != "tokio" ]; then
+    echo "invalid async runtime: ${ASYNC_RUNTIME}" >&2
+    exit 1
+fi
+
+. ~/.cargo/env
```

Comment:
I think this should get run as part of `env.sh` and isn't required here.

Reply:
Done.